bin/cohere-embed-v3-services.ts

```diff
@@ -33,12 +34,13 @@ const cohereEmbedModel = process.env.COHERE_EMBED_MODEL!;
 const imageVersion = process.env.IMAGE_VERSION!;
 const ecrRepositoryName = process.env.ECR_REPOSITORY_NAME!;
 const dockerfileName = process.env.DOCKERFILE_NAME!;
-const port = process.env.PORT!;
+const cdkDeployPort = process.env.PORT!;
```

Comment:
**Suggestion:** Consider using a more descriptive variable name instead of `cdkDeployPort` to enhance code readability and maintainability.
> 🚁  **Medium Priority** 
 > [category: **`maintainability`**, importance: **`7`**]
> **Reason:** The suggestion to rename `cdkDeployPort` to `deployPort` enhances code readability and maintainability by using a more descriptive variable name.
 

```suggestion
37 +const deployPort = process.env.PORT!;
```